frontend/src/containers/Loading.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import LoadingComponent from '../components/Loading'
+
+const nextPosition = (position, max, direction) => {
+  if (direction < 0 && position <= 0) return 1
+  if (direction > 0 && position >= max - 1) return max - 2
+  return position + direction
```

Comment:
```bash
> nextPosition(1,10,1)
2
> nextPosition(10,10,1)
8
> nextPosition(2,10,1)
3
> nextPosition(2,10,-1)
1
> nextPosition(1,10,-1)
0
> nextPosition(0,10,-1)
1
> nextPosition(10,10,-1)
9
```

I think that some results are weird... Like `8` from params `10, 10, 1` (should be 10?) and `1` from params `0,10,-1` (should be 0?)
I've thought on this code below, check if it helps:
```js
const nextPosition = (position, max, direction) => Math.min(Math.max(position+direction, 0), max);
```
And here are the results:
```bash
> nextPosition(1,10,1)
2
> nextPosition(10,10,1)
10
> nextPosition(2,10,1)
3
> nextPosition(2,10,-1)
1
> nextPosition(1,10,-1)
0
> nextPosition(0,10,-1)
0
> nextPosition(10,10,-1)
9
```